core/state/statedb.go

```diff
@@ -1130,6 +1146,6 @@ func (s *StateDB) SlotInAccessList(addr common.Address, slot common.Hash) (addre
 	return s.accessList.Contains(addr, slot)
 }
 
-func (s *StateDB) GetSubstatePostAlloc() substate.SubstateAlloc {
-	return s.SubstatePostAlloc
+func (s *StateDB) GetPostWorldState() substate.WorldState {
+	return s.PostWorldState
```

Comment:
Last commit (902a48c96826b6eb569edf1673ecf8aad135d0be) makes `s.PostWorldState` always nil.
Previous commit (00dfa731271171914ed7ce71264198b3f8875baf) works.

Reply:
Mb, I forgot to call `EnableRecordReplay()`. Everything works.